src/Cultuurkuur/data/regions.json

```diff
@@ -4,124 +4,124 @@
             "nl": "Brussels Hoofdstedelijk Gewest"
         },
         "label": "cultuurkuur_werkingsregio_provincie_nis-01000",
+        "extraLabel": "cultuurkuur_werkingsregio_nis-01000",
```

Comment:
Looks fine, but maybe also check with @vhande because, this will almost exclusivly be used by the frontEnd.